rpmvenv/extensions/python/venv.py

```diff
@@ -108,18 +108,23 @@ def generate(config, spec):
         spec.blocks.files.append('/%{venv_install_dir}')
 
         spec.blocks.install.append('%{venv_cmd} %{venv_dir}')
-        for requirement in config.python_venv.requirements:
-
-            spec.blocks.install.append(
-                '%{{venv_pip}} -r %{{SOURCE0}}/{0}'.format(
-                    requirement,
+        # If we have requirements files, install based on them.
+        if len(tuple(config.python_venv.requirements)) > 0:
```

Comment:
This also does not change the behaviour of the code. Looping over an empty iterable already results in the loop body never getting executed.